service/build.gradle.kts

```diff
@@ -82,15 +83,15 @@ dependencies {
 
     listOf(
         libs.hamkrest,
-        libs.mockk,
-        libs.kotlin.faker,
-        libs.kotlin.coroutines.test,
-        libs.spring.mockk,
         libs.kotest,
         libs.kotest.assertions,
         libs.kotest.assertions.arrow,
         libs.kotest.property,
         libs.kotest.spring,
+        libs.kotlin.coroutines.test,
+        libs.kotlin.faker,
+        libs.mockk,
+        libs.spring.mockk,
```

Comment:
Aside from the Kotlin & Java version bumps, the only changes to this file are from sorting of the dependency lists